bofire/kernels/categorical.py

```diff
@@ -0,0 +1,25 @@
+import torch
+from gpytorch.kernels.kernel import Kernel
+from torch import Tensor
+
+
+class HammingKernelWithOneHots(Kernel):
```

Comment:
Can you add a docstring which refers to the botorch implementation and just states that this is the same but based on a one-hot representation?